openpype/hosts/houdini/plugins/publish/validate_review_colorspace.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import PublishValidationError
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.houdini.api.action import SelectROPAction
+
+
+class SetDefaultViewSpaceAction(RepairAction):
+    label = "Set default view space"
+    icon = "mdi.monitor"
+
+
+class ValidateReviewColorspace(pyblish.api.InstancePlugin):
+    """Validate Review Colorspace parameters.
+
+    It checks if 'OCIO Colorspace' parameter was set to valid value.
+    """
+
+    order = pyblish.api.ValidatorOrder + 0.1
+    families = ["review"]
+    hosts = ["houdini"]
+    label = "Validate Review Colorspace"
+    actions = [SetDefaultViewSpaceAction, SelectROPAction]
+
+    def process(self, instance):
+        invalid = self.get_invalid(instance)
+        if invalid:
+            raise PublishValidationError(
+                ("'OCIO Colorspace' parameter is not valid."),
+                title=self.label
+            )
+
+    @classmethod
+    def get_invalid(cls, instance):
+        import hou  # noqa
+        import os
```

Comment:
Why are these imports here and not at the top?

Reply:
because I found import done in this way in many plugins 
my bad, I should have asked about the better way.